tests/api.c

```diff
@@ -314,6 +314,44 @@ static void test_wolfSSL_CTX_use_PrivateKey_file(void)
 }
 
 
+static void test_wolfSSL_get_privateKey(void)
+{
+#if defined(OPENSSL_EXTRA)
+#if !defined(NO_FILESYSTEM) && !defined(NO_CERTS) && !defined(NO_RSA)
+
+    WOLFSSL_CTX *ctx;
+    WOLFSSL *ssl;
+
+    AssertNotNull(ctx = wolfSSL_CTX_new(wolfSSLv23_server_method()));
+
+    /* test for if key owned */
+    AssertTrue(wolfSSL_CTX_use_certificate_file(ctx, svrCertFile, SSL_FILETYPE_PEM));
+    AssertTrue(wolfSSL_CTX_use_PrivateKey_file(ctx, svrKeyFile, SSL_FILETYPE_PEM));
+    AssertNotNull(ssl = wolfSSL_new(ctx));
+
+    assert(wolfSSL_get_privatekey(ssl) != NULL);
+
+    wolfSSL_free(ssl);
+    wolfSSL_CTX_free(ctx);
+
+      /* test for if not key owned */
+    AssertNotNull(ctx = wolfSSL_CTX_new(wolfSSLv23_server_method()));
+    AssertTrue(wolfSSL_CTX_use_certificate_buffer(ctx, server_cert_der_2048,
+                sizeof_server_cert_der_2048, SSL_FILETYPE_ASN1));
+    AssertTrue(wolfSSL_CTX_use_PrivateKey_buffer(ctx, server_key_der_2048,
+                sizeof_server_key_der_2048, SSL_FILETYPE_ASN1));
+    AssertNotNull(ssl = wolfSSL_new(ctx));
+
+    assert(wolfSSL_get_privatekey(ssl) != NULL);
```

Comment:
Same as above, it would be helpful to test that this is returning what we expect.